src/main/api/deployer-api.yaml

```diff
@@ -684,3 +684,14 @@ components:
               message:
                 type: string
                 example: OK
+
+    Created:
```

Comment:
What's this?

Reply:
It was missing a `Created` response when the duplicate target API was added https://github.com/craftercms/deployer/commit/7776d676acaae04d4c070187da6a5815ebf5ddd6